### PR TITLE
Ruby env for windows

### PR DESCRIPTION
**Pull request by manuGil**
Hello, 

In case it might be usefull for the community, I added a list of instructions to compile the pages  locally using Ruby on Windows.

### CODE REVIEW

**Comment by emmyft** [comment]
Thank you so much @manuGil 😍 

hmmm I don't have the technical abilities to review this PR- @mwakok / @sebranchett could you please help, maybe? Thank you!!

**Comment by sebranchett** [comment]
Hi @emmyft @mwakok @manuGil , My laptop has just been reinstalled and I am going to pick it up tomorrow (Friday 6 November). An ideal opportunity to test the ruby instructions. I'll let you know either tomorrow, or Tuesday, depending on how many other problems I have with my laptop. :)

**Comment by emmyft** [comment]
sounds like a great plan @sebranchett (: fingers crossed for a smooth reinstallation, keep us posted! 🤞

**Comment by sebranchett** [comment]
Great job @manuGil. The website is now running on my Windows 10 laptop!

One suggestion and 2 spelling mistakes:

I think we should change the title 'Using miniconda' to 'Using Ruby on linux/macOS'
and make the title 'Run the link checks' smaller (#### instead of ###).
Hopefully this will make it obvious that you either use make or follow the Windows instructions.

Step 1:
Mind about a 32-bit or 34-bit.
34 should be 64-bit.

Step 2:
When done, press enter once more. hel
hel can be removed. :)

File: README.md [hunk 1]
```diff
@@ -71,11 +71,11 @@ $ make check-html
 
 We asume you have a working installation of [Git for Windows](https://git-scm.com/downloads)
 
-1. Download and install the latest version of `Ruby+Devkit` for Windows. Keep all the default options. Mind about `x64` or `x86`: https://rubyinstaller.org/downloads. Put attention to where the location of the root directory. For example, `C:\Ruby27-x64`
+1. Download and install the latest version of [Ruby+Devkit for Windows](https://rubyinstaller.org/downloads). Mind about a *32-bit*  or *34-bit*.   Keep all the default options. Put attention to where the location of the Ruby's root directory is located. For example, `C:\Ruby27-x64`
```
Comment (emmyft) [inline README.md:74]: 64-bit instead of 34-bit (:

File: README.md [hunk 2]
```diff
@@ -71,11 +71,11 @@ $ make check-html
...
 
 2. When the installation is done a *command prompt*  will pop up. Type `3` and press enter. This will install the Ruby's toolchain. When done, press enter once more.
-
-3. Install `Jekyll` and `Bundler`. On the powershell run:
+hel
```
Comment (emmyft) [inline README.md:77]: remove "hel"

File: README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ you know how to install [Bundler](https://bundler.io/) and
 that can install all these tools for you. You can install it by following the
 instructions on this page: https://conda.io/docs/user-guide/install/index.html
 
-In the sequel, we assume you use miniconda.
+### Using miniconda
```
Comment (emmyft) [inline README.md:31]: @sebranchett: we should change the title 'Using miniconda' to 'Using Ruby on linux/macOS'
+1 from me (:

File: README.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ instructions on this page: https://conda.io/docs/user-guide/install/index.html
 7. Open the website in your favorite browser at:
    [http://127.0.0.1:4000/](http://127.0.0.1:4000/)
 
-## Run the link checks
+### Run the link checks
```
Comment (emmyft) [inline README.md:62]: heading 4 instead? i.e. #### Run the link checks

**Comment by emmyft** [comment]
Thank you so much @sebranchett (: 

@manuGil I've converted Susan's comments into in-code comments... please address, and then we can close this? 

I've just followed this on my windows machine and it works like a charm! Thank you so much both for your work 🙏

File: README.md (hunk 3 above)
Comment (manuGil) [reply README.md:31]: I think the change make sense, and I applied it.

**Comment by manuGil** [comment]
@emmyft I made the changes based on your and @sebranchett  comments. You can merge the branch and delete the branch.

**Comment by emmyft** [comment]
Thank you very much @manuGil and @sebranchett (: If it's OK, I'd love to share this back with upstream

**Pull request merged**